File: output/ChatGPT/avatar/Java/C++/codeforces_268_A.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    cin >> n;
    int result = 0;
    vector<int> h(n);
    vector<int> g(n);

    for (int i = 0; i < n; i++) {
        cin >> h[i];
        cin >> g[i];
    }

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            if (h[i] == g[j]) {
                result++;
            }
        }
    }

    cout << result << endl;

}

File: output/ChatGPT/avatar/Java/Python/codeforces_268_A.py
n = int(input())
result = 0
h = []
g = []

for i in range(n):
    h_val, g_val = map(int, input().split())
    h.append(h_val)
    g.append(g_val)

for i in range(n):
    for j in range(n):
        if h[i] == g[j]:
            result += 1

print(result)